Javascript/10 exercicios/02-fluxo-caixa-familiar.js

```diff
@@ -42,11 +42,12 @@ function familyBalance() {
   
   if(result < 0) {
     console.log(`O saldo negativo é de R$${result}`);
-  } else if(result === 0) {
+  } if(result === 0) {
     console.log(`Os gastos igualam as despesas. R$${result}`);
-  } else {
+  } {
     console.log(`O saldo positivo é de R$${result}`);
   }
```

Comment:
você teria que retornar o resultado para remover o `else`. exemplo:

```javascript
if (condition1) {
  return ...
}

if (condition2) {
  return ...
}

// in case none of the condition is true, return a fallback value
return ...

Reply:
pesquisa sobre "early return pattern"

Reply:
Pesquisei um pouco sobre early return pattern e usei o seu exemplo para implementar. No caso, usei o console.log na chamada da função para que aparecesse o resultado no terminal. Aguardo instruções!

Reply:
aqui ta certo! dá uma olhada no [outro comentário](https://github.com/tomsantos07/trilha-fundamentar/pull/27#discussion_r1108485726)